packages/components/src/styles/global.js

```diff
@@ -0,0 +1,32 @@
+import { GlobalStyles } from "twin.macro";
+import React from "react";
+import { createGlobalStyle } from "styled-components";
+
+const GlobalFonts = createGlobalStyle`
```

Comment:
I was inspired by https://dev.to/alaskaa/how-to-import-a-web-font-into-your-react-app-with-styled-components-4-1dni

Please let me know if this is a bad idea!

Reply:
I don't think it's a bad idea, but thinking about it more, we may want to add other ways to import it since these are gonna stay basically forever. Either that, or to include the css variables from tokens into it somehow.

Reply:
ah yeah exporting the css variables would be good